fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/CardNudge.kt

```diff
@@ -0,0 +1,281 @@
+package com.microsoft.fluentui.tokenized.notification
+
+import androidx.compose.animation.core.TweenSpec
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.gestures.Orientation
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material.ripple.rememberRipple
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.IntOffset
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.compose.FractionalThreshold
+import com.microsoft.fluentui.compose.rememberSwipeableState
+import com.microsoft.fluentui.compose.swipeable
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.FluentTheme.aliasTokens
+import com.microsoft.fluentui.theme.token.*
+import com.microsoft.fluentui.theme.token.controlTokens.CardNudgeInfo
+import com.microsoft.fluentui.theme.token.controlTokens.CardNudgeTokens
+import com.microsoft.fluentui.theme.token.controlTokens.PillButtonInfo
+import com.microsoft.fluentui.theme.token.controlTokens.PillButtonTokens
+import com.microsoft.fluentui.tokenized.segmentedcontrols.PillButton
+import com.microsoft.fluentui.tokenized.segmentedcontrols.PillMetaData
+import kotlin.math.roundToInt
+import androidx.compose.material.Icon as MaterialIcon
+
+private val LocalCardNudgeTokens = compositionLocalOf { CardNudgeTokens() }
+private val LocalCardNudgeInfo = compositionLocalOf { CardNudgeInfo() }
+
+// TAGS FOR TESTING
+private val CARDNUDGE = "CardNudge"
```

Comment:
Check if the same tag could be directly refer in UITestCases

Reply:
As discussed, need another round of discussion. Will do this in another PR.